_data/harnesses/multiple-event-types/confluent.yml

```diff
@@ -0,0 +1,213 @@
+dev:
+  steps:
+
+    - title: Provision your Kafka cluster
+      content:
+        - action: skip
+          render:
+            file: shared/markup/ccloud/ccloud-setup-self.adoc
+
+    - title: Initialize the project
+      content:
+        - action: execute
+          file: tutorial-steps/dev/init.sh
+          render:
+            file: tutorials/multiple-event-types/kafka/markup/dev/init.adoc
+
+        - action: execute
+          file: tutorial-steps/dev/make-configuration-dir.sh
+          render:
+            file: tutorials/multiple-event-types/kafka/markup/dev/make-config-dir.adoc        
+
+    - title: Write the cluster information into a local file
+      content:
+        - action: skip
+          render:
+            file: shared/markup/ccloud/config-create.adoc
+
+    - title: Download and setup the Confluent Cloud CLI
```

Comment:
It's now called just Confluent CLI.  Should this PR make that change, or is there another PR for that change?

Reply:
ack